internal/pkg/manifest/validate.go

```diff
@@ -660,6 +682,9 @@ func (e *EFSVolumeConfiguration) Validate() error {
 			conditionalFields: []string{"gid"},
 		}
 	}
+	if e.UID != nil && *e.UID == 0 {
```

Comment:
nit: what do you think of 
```suggestion
	if aws.Uint32Value(e.UID) {
```
The validation for `TopicSubscription`'s `Name` field also seems to be taking advantage of this function and the fact that it returns a zero value if the pointer is `nil`.

Reply:
They are not quite the same...for the former if `e.UID` is nil then we won't do the validation and let it go, while the latter we'll throw an error which we shouldn't